Pod/Classes/TintedButton/TintedButton.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  TintedButton.swift
+//  Pods
+//
+//  Created by Michael Skiba on 11/17/16.
+//
+// Copyright 2016 Raizlabs and other contributors
+// http://raizlabs.com/
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import UIKit
+
+/// A button that looks and behaves like the default "Get" button in iTunes
+/// To fully replacate the effect in views built in interface builder, the button type needs to be set to "custom"
+open class TintedButton: UIButton {
+
+    @IBInspectable open var rz_backgroundColor: UIColor = UIColor.clear {
```

Comment:
#? I wonder if there's a way to use the button states and a overrides of `backgroundColor` and `tintColor` to streamline the API and keep parity with other UIViews? I say I wonder because a) I'm not that there is and b) it might be more confusing than helpful. I like how straightforward this is, but the `rz_` bugs me.